force-app/tdtm/triggerHandlers/account/ACCT_Accounts_TDTM.cls

```diff
@@ -61,7 +61,7 @@ public class ACCT_Accounts_TDTM extends TDTM_Runnable {
             when BeforeUpdate {
                 if (!isContactBeforeInsertContext()) {
                     AccountAdapter accountsAdapter = new AccountAdapter(listNew, listOld);
-                    accountsAdapter.onBeforeUpdate();
+                    accountsAdapter.onBeforeUpdate_ACCT_Accounts_TDTM();
```

Comment:
I know we named these methods to differentiate the action that triggered these updates but I would love to see us use the `onBeforeUpdate` method name and maybe fork the behaviour we want in the `AccountAdapter` class since we already have guard methods on the `onBeforeUpdate_ADDR_Account_TDTM` method to determine if we run its containing logic or not.

Reply:
When there is only a single trigger handler per object, then the logic will be able to be combined into a single onBefore... method.  For now these are named this way to clarify that there are multiple handlers running in the same context on the same objects.